StepDefinitions/TestSteps.cs

```diff
@@ -0,0 +1,131 @@
+using System;
+using TechTalk.SpecFlow;
+
+namespace TestSpecs {
+    [Binding]
+    class TestStpes
+    {
+        [Given(@"I am on the (.*) website")]
+        public void GivenIAmOnTheNLWebsite(string countryCode)
```

Comment:
* I don't suggest you put `Given`/`When`/`Then` as the function name,  because one function could have multiple bindings, also you could bind a function as a **GENERAL** function